### DISCUSSION TITLE
Toggle on/off with a keyboard shortcut

### DISCUSSION THREAD

**Discussion by kristjanmar** [moalamri/vscode-inline-fold#44] [Ideas]
Thanks for this extension! It is something I've wanted for a very long time to make code with Tailwind look cleaner but I only found out that this exists about a week ago.

I have one feature suggestion that I think would make it even better. How about a simple keyboard shortcut to toggle the folding on/off? For example, CTRL+ALT+F (F for fold) or anything else that is easy to toggle.

When the main reason I am editing a file is due to styling reasons, then it's a bit disruptive having to click each line with styles in order to see them. A simple keyboard shortcut to toggle this on/off would make it easier instead of having to manually disable the extension and then re-enable it afterwards when I'm done editing or refactoring the styling.

Also, you might want to change the title on the VS Code extension page to something like "Inline fold - Hide CSS and Tailwind classes" -- that way it will be easier to find it when searching for the extension. Because I'm sure a lot of people are looking for this.

**Comment by moalamri** [comment]
Hey @kristjanmar 
I'm glad you found it useful :) 
The toggle shortcut exists already with no default keybinding. Search for Inline Fold in the commands panel and you can set the desired keys.

As for the extension title I left it generic in purpose, there seems to be other use cases especially with markdown 👍
If I were to do a pool and find the high percentage of use case is with Tailwindcss then yea probably would make it serve that purpose with less settings, and make an extended version for any other use case 🤔

All for future. As for now I'm pretty much fully occupied by work

**Reply by kristjanmar** [reply]
Fantastic, thanks! I added your instructions to the README via pull request if you are interested in including it. I think many people will want to know about this.

**Reply by moalamri** [reply]
Thank you 👍
